modules/moordyn/src/MoorDyn.f90

```diff
@@ -163,7 +163,7 @@ SUBROUTINE MD_Init(InitInp, u, p, x, xd, z, other, y, m, DTcoupling, InitOut, Er
       CALL DispNVD( MD_ProgDesc )
       InitOut%Ver = MD_ProgDesc
 
-      CALL WrScr('   This is an alpha version of MoorDyn-F v2, with significant input file changes from v1.')  
+      CALL WrScr('   This is MoorDyn v2, with significant input file changes from v1.')
```

Comment:
This might conflict with #1371 

(test comment to see why comments aren't showing up properly)